docs_user/assemblies/assembly_red-hat-ceph-storage-prerequisites.adoc

```diff
@@ -0,0 +1,30 @@
+ifdef::context[:parent-context: {context}]
+
+[id="red-hat-ceph-storage-prerequisites_{context}"]
+
+:context: ceph-prerequisites
+
+= {Ceph} prerequisites
+
+Before you migrate your {Ceph} cluster to the {rhos_long_noacro} {rhos_curr_ver} environment, complete the following tasks in your {rhos_prev_long} {rhos_prev_ver} environment:
```

Comment:
Technically we're not migrating Ceph to RHOSO, we're "externalizing the Ceph cluster", but no service is "podified". I think this sentence might generate confusion in customers' expectations: can we change it to something like: "Before migrating your {Ceph} cluster daemons from the Controller nodes, ensure the following tasks are completed in your {rhos_prev_long} {rhos_prev_ver} environment:"

Also, can we make a note about "External Ceph"?

```
If NFS is part of the OpenStack deployment we should  recreate a ceph-nfs cluster in the target nodes, other than propagate the StorageNFS network;
```

Of course Ceph must be upgraded first to Ceph 7 in both cases.

Reply:
I think we also need to move this section [1] to prerequisites, that is related to my previous comment.

[1] https://logserver.rdoproject.org/60/660/deddc62cd5b61de99d7e312dd230181557749d77/github-check/adoption-docs-preview/22fd464/docs_build/docs_build/adoption-user/index-downstream.html#creating-a-ceph-nfs-cluster_migrating-databases

Reply:
Do you want me to add that new "external Ceph" prerequisite to [1], rather than to the top-level Ceph prerequisites?

I would also rewrite it slightly to clarify that it applies to external Ceph deployments, e.g., "If you use external Ceph, you must recreate a `ceph-nfs` cluster."

Reply:
I think we can include the "external Ceph" in the top-level Ceph prerequisites, the `ceph-nfs` creation would just represent a common step.
I think (and we learned it w/ 17 FFU doc), it's better to clarify in advance the distinction between external Ceph and internal Ceph, and separate the use cases.

Reply:
I also need a clarification on this part: " recreate a ceph-nfs cluster in the target nodes, other than propagate the StorageNFS network;"

Does "other than" mean "instead of"? Or "as well as"?